src/Form/WebdamConfig.php

```diff
@@ -33,6 +66,11 @@ protected function getEditableConfigNames() {
    */
   public function buildForm(array $form, FormStateInterface $form_state) {
     $config = $this->config('media_webdam.settings');
+    $webdamFolders = [];
+    // $webdamFolders = \Drupal::service('media_webdam.admin_settings')->folderList();
+    if (!empty($config->get('username')) && !empty($config->get('password'))) {
```

Comment:
This conditional isn't needed. If credentials aren't provided, the Webdam client will throw an error.